### PR TITLE
Bump threejs to version 0.153.0

### PR DESCRIPTION
**Pull request by NataliaDSmirnova**
Update threejs version to 0.153.0 version

## Type of changes

- New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read [CONTRIBUTING](https://github.com/epam/miew/blob/master/CONTRIBUTING.md) and [CODE_OF_CONDUCT](https://github.com/epam/miew/blob/master/CODE_OF_CONDUCT.md) guides.
- [x] I have followed the code style of this project.
- [x] I have run `yarn run ci`: lint and tests pass locally with my changes.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] I have added necessary documentation / The changes do not need docs update.

### CODE REVIEW

**Review by DmitryLukyanov — DISMISSED** [review]
LGTM. Nice work!
Checked that all "0.125" tokens in the solution have been replaced.

File: packages/lib/src/gfx/shaders/UberMaterial.js [hunk 1]
```diff
@@ -73,84 +73,8 @@ const uberOptionNames = [
   'srcTexelSize',
 ];
 
-function UberMaterial(params) {
```
Comment (paulsmirnov) [inline packages/lib/src/gfx/shaders/UberMaterial.js:76]: This diff is difficult to follow. I hope you included the fix I created for a similar change in the `master` branch. There was a not quite correct migration to classes.

File: packages/lib/test/e2e/golden/1aid_demo.png
Comment (paulsmirnov) [inline packages/lib/test/e2e/golden/1aid_demo.png:1]: The diff is in axes, right?

**Review by paulsmirnov — COMMENTED** [review]
Awesome. Could you answer a few comments?

**Comment by NataliaDSmirnova** [comment]
> LGTM. Nice work! Checked that all "0.125" tokens in the solution have been replaced.
Good idea. Done!

File: packages/app/package.json [hunk 2]
```diff
@@ -72,6 +72,7 @@
     "redux": "^4.0.5",
     "rimraf": "^3.0.2",
     "style-loader": "^3.3.3",
+    "three": "0.153.0",
```
[the remark this answers is not in the document]
Comment (NataliaDSmirnova) [reply packages/app/package.json:75]: I guess, threejs is a peer dependency for miew.
Will it be installed for app if not set explicitly in app?

File: packages/lib/src/gfx/shaders/Uber.frag [hunk 3]
```diff
@@ -447,7 +446,7 @@ float unpackRGBAToDepth( const in vec4 v ) {
       }
     #endif
 
-    vec3 irradiance = dotNL * directLight.color * PI;
+    vec3 irradiance = dotNL * directLight.color;
```
[the remark this answers is not in the document]
Comment (NataliaDSmirnova) [reply packages/lib/src/gfx/shaders/Uber.frag:449]: They moved the multiplication from shaders into light sources math. So we just support the change in our shader.

File: packages/lib/src/gfx/shaders/UberMaterial.js (hunk 1 above)
Comment (NataliaDSmirnova) [reply packages/lib/src/gfx/shaders/UberMaterial.js:76]: When I made the change I followed your fix in the `master` branch.

File: packages/lib/src/Miew.js [hunk 4]
```diff
@@ -57,6 +57,10 @@ const EDIT_MODE = { COMPLEX: 0, COMPONENT: 1, FRAGMENT: 2 };
 const LOADER_NOT_FOUND = 'Could not find suitable loader for this source';
 const PARSER_NOT_FOUND = 'Could not find suitable parser for this source';
 
+// Color management changed a lot in threejs 152+ version.
+// To store miew colors we disable the new color management system
```
[the remark this answers is not in the document]
Comment (NataliaDSmirnova) [reply packages/lib/src/Miew.js]: Fixed

File: packages/app/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (paulsmirnov) [reply packages/app/package.json:75]: It must (and it does). I believe we don't need to mention it in peer dependencies until we use it ourselves (e.g., call THREE methods in the app). As I said, I removed `three` in #500 and it builds and works perfectly.

**Review by paulsmirnov — CHANGES_REQUESTED** [review]
Please remove `three` from the app's dependencies and verify that it works (it worked for me). It is an implicit dependency as we don't use it directly.

**Review by paulsmirnov — APPROVED** [review]
Hooray! Thank you 🥳

**Pull request merged**